install/helm/agones/values.yaml

```diff
@@ -137,6 +137,7 @@ agones:
       expose: true
       response: ok
       port: 80
+      nodePort: 0 # nodePort will be used if the serviceType is set to NodePort
```

Comment:
Please add documentation in https://github.com/googleforgames/agones/blob/main/site/content/en/docs/Installation/Install%20Agones/helm.md

Reply:
Going to get this in, then I'll handle this after this is merged.